doc/en/explanation/pythonpath.rst

```diff
@@ -8,15 +8,15 @@ pytest import mechanisms and ``sys.path``/``PYTHONPATH``
 Import modes
 ------------
 
-pytest as a testing framework needs to import test modules and ``conftest.py`` files for execution.
+Pytest as a testing framework that needs to import test modules and ``conftest.py`` files for execution.
```

Comment:
pytest is actually all lower-case, I'm not sure the proper grammar in that case for the beginning of sentences.

```suggestion
pytest as a testing framework that needs to import test modules and ``conftest.py`` files for execution.
```